lib/Traits/WebFeatureServiceCatalogItemTraits.ts

```diff
@@ -0,0 +1,51 @@
+import { JsonObject } from "../Core/Json";
+import anyTrait from "./anyTrait";
+import CatalogMemberTraits from "./CatalogMemberTraits";
+import FeatureInfoTraits from "./FeatureInfoTraits";
+import GetCapabilitiesTraits from "./GetCapabilitiesTraits";
+import LayerOrderingTraits from "./LayerOrderingTraits";
+import MappableTraits from "./MappableTraits";
+import mixTraits from "./mixTraits";
+import primitiveTrait from "./primitiveTrait";
+import SplitterTraits from "./SplitterTraits";
+import UrlTraits from "./UrlTraits";
+import objectTrait from "./objectTrait";
+import { StyleTraits } from "./GeoJsonCatalogItemTraits";
+
+export default class WebFeatureServiceCatalogItemTraits extends mixTraits(
+  FeatureInfoTraits,
+  LayerOrderingTraits,
+  SplitterTraits,
+  GetCapabilitiesTraits,
+  UrlTraits,
+  MappableTraits,
+  CatalogMemberTraits
+) {
+  @primitiveTrait({
+    type: "string",
+    name: "Type Name(s)",
+    description: "The type name or names to display."
+  })
+  typeNames?: string;
```

Comment:
Should we allow multiple layers in a single `WebFeatureServiceCatalogItem` do you think? I know this is currently ported from how we do this in Terria master... but should we re-evaluate?
I'm trying to think when it would be helpful to have multiple layers represented as a single layer in Terria...?

Reply:
I would say may as well leave it in, so people could add operations using the `parameters` trait if they really want to.
But yeah, probably not very helpful - I haven't ever seen it used.